content/en/account_management/teams.md

```diff
@@ -76,14 +68,23 @@ Team handles that aren't associated with a defined team in Datadog behave simila
 
 Datadog supports associating the following resources with team handles:
 
-- [Dashboards][3]
-- [Incidents][4]
-- [Monitors][5]
-- [Resource Catalog][6]
-- [Service Catalog][7]
-- [Service Level Objectives][8]
+- [Dashboards][4]
+- [Incidents][5]
+- [Monitors][6]
+- [Resource Catalog][7]
+- [Service Catalog][8]
+- [Service Level Objectives][9]
 - Synthetic Tests, Global Variables, Private Locations
 
+### Send notifications to a separate communication channel 
```

Comment:
Hey @estherk15,
I'm not sure "separate" is the right word here? Maybe "specific" would work better? WDYT?

Reply:
```suggestion
### Send notifications to a specific communication channel 
```